kinto/__main__.py

```diff
@@ -9,7 +9,7 @@
 
 from kinto.config import init
 
-CONFIG_FILE = 'config/kinto.ini'
+CONFIG_FILE = 'kinto/config/kinto.ini'
```

Comment:
I believe that's actually the behaviour we want: the "kinto" folder is the one where all the kinto files are contained, and you shouldn't temper it. Rather, creating a "config" folder where you run the command seems to make more sense.